tool/config/linkcheck-skip-list.txt

```diff
@@ -3,3 +3,11 @@
 # FIXME(Temporary): linkcheck seems to be intermittently failing when accessing main.css
 # https://github.com/dart-lang/site-www/issues/1107
 /assets/main(-[0-9a-f]+)?.css
+https://localhost:4040
```

Comment:
Do we want to use a wildcard to skip all localhost links?

Reply:
I don’t think so, because the link checker uses localhost when it checks links.